src/etc/inc/upgrade_config.inc

```diff
@@ -5493,6 +5493,22 @@ function upgrade_172_to_173() {
 	}
 }
 
+/*
+ * Automatically enable retrieving captive portal bandwidth limits from RADIUS for each captive portal
+ * instance already enabled
+ */
+function upgrade_173_to_174() {
```

Comment:
This is out of date. Please check the current latest_config version and update accordingly. I think you will need 175_to_176, but please check immediately before pushing the change.